pkg/utils/image.go

```diff
@@ -37,18 +38,22 @@ func getEtcdImageKeys(useEtcdWrapper bool) (etcdImageKey string, etcdbrImageKey
 // It will give preference to images that are set in the etcd spec and only if the image is not found in it should
 // it be picked up from the image vector if it's set there.
 // A return value of nil for either of the images indicates that the image is not set.
-func GetEtcdImages(etcd *druidv1alpha1.Etcd, iv imagevector.ImageVector, useEtcdWrapper bool) (*string, *string, error) {
-	etcdImageKey, etcdbrImageKey := getEtcdImageKeys(useEtcdWrapper)
+func GetEtcdImages(etcd *druidv1alpha1.Etcd, iv imagevector.ImageVector, useEtcdWrapper bool) (*string, *string, *string, error) {
+	etcdImageKey, etcdbrImageKey, alpineImageKey := getEtcdImageKeys(useEtcdWrapper)
```

Comment:
also change it here. Let us just use one identifier, so this should also be `initContainerImageKey`